### PR TITLE
Fix the class name for the rng box

### PR DESCRIPTION
**Pull request by samuellim-exe**
I think blooket changed the top bar class name. So, im here to propose this change.

### CODE REVIEW

**Comment by person4268** [comment]
So that actually isn't the same element. The actual element is `styles__headerRight___1R77a-camelCase` or something like that. While putting it there does work, a) it would make more sense to get it by id as it doesn't change and b) it breaks the positioning of the login button

File: blooket.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function rigRngBox() {
 	input.step = 0.001;
 	input.value = 0.5;
 	input.id = "lolinput";
-	document.getElementsByClassName("styles__headerRight___1bkCl-camelCase")[0].appendChild(input);
+	document.getElementsByClassName("styles__header___25QYr-camelCase")[0].appendChild(input);
```
Comment (person4268) [inline blooket.js]:
```suggestion
	document.getElementsByClassName("styles__headerRight___1R77a-camelCase")[0].appendChild(input);
```

**Comment by person4268** [comment]
I was experimenting a little bit but i'll merge it now (with the correct class name)

**Comment by samuellim-exe** [comment]
Ok.....

**Pull request merged**